controller/planner_backlog.go

```diff
@@ -74,39 +112,25 @@ func (c *PlannerBacklogController) List(ctx *app.ListPlannerBacklogContext) erro
 			}
 			exp = criteria.And(exp, expWits)
 		} else {
-			// If there isn't work item types, return an empty array.
-			return ctx.OK(&app.WorkItemList{
-				Data:  []*app.WorkItem{},
-				Links: &app.PagingLinks{},
-				Meta:  &app.WorkItemListResponseMeta{TotalCount: 0},
-			})
+			// If there isn't work item types, it'll return an empty array.
+			return nil
 		}
 
 		// Get the list of work items for the following criteria
-		result, tc, err := appl.WorkItems().List(ctx.Context, spaceID, exp, &offset, &limit)
-		count := int(tc)
+		var tc uint64
+		result, tc, err = appl.WorkItems().List(ctx, spaceID, exp, offset, limit)
```

Comment:
Could we look at adding a Count(criteria) or something.. returning the first page from db is just wasted resources.